src/PKSim.Core/Snapshots/CompoundProcess.cs

```diff
@@ -1,9 +1,15 @@
-﻿namespace PKSim.Core.Snapshots
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace PKSim.Core.Snapshots
 {
    public class CompoundProcess : ParameterContainerSnapshotBase
    {
+      [Required]
```

Comment:
This DataAnnotation indicates which parameters will be seeen as required when generating the schema